src/runtest.rs

```diff
@@ -2649,3 +2650,25 @@ fn read2_abbreviated(mut child: Child) -> io::Result<Output> {
         stderr: stderr.into_bytes(),
     })
 }
+
+// FIXME: Remove this when rotate_left is stable in 1.26
+#[cfg(feature = "norustc")]
+fn read2_abbreviated(mut child: Child) -> io::Result<Output> {
```

Comment:
This looks quite different from the standard version of `read2_abbreviated()`, why is that?

Reply:
This is just because I don't want to copy the `slice::rotate_left` impl for now(It should be done if we are going to merge this), see the FIXME above.

Reply:
I was just wondering why `read2_abbreviated()` was affected by the `norustc` feature, when it's because of `rotate_left`. I'd rather have an:

```
// FIXME: Remove this when rotate_left is stable in 1.26
#[cfg(feature = "norustc")]
fn rotate_left(..) -> {
    ...
}
```
to switch implementations of `rotate_left`.